motor_control_config:
    enable:
        enable: false
    parameters:
        acceleration_limit: 0.0
        low_batt_th: 0.0
        max_temperature: 0.0
        mode: 0
        motor_encoder_steps_per_revolution: 0
        potentiometer_gain: 0.0
        second_encoder_steps_per_revolution: 0
        thermal_capacity: 0.0
        thermal_current_gain: 0.0
        thermal_resistance: 0.0
        torque_constant: 0.0
        torque_limit: 0.0
        transmission_ratio_p: 0
        transmission_ratio_q: 0
        velocity_limit: 0.0
    pid_current:
        d: 0.0
        i: 0.0
        i_limit: 0.0
        p: 0.0
    pid_position:
        d: 0.0
        i: 0.0
        i_limit: 0.0
        p: 0.0
    pid_velocity:
        d: 0.0
        i: 0.0
        i_limit: 0.0
        p: 0.0
    stream:
        current_pid: 0
        index: 0
        motor_encoder: 0
        motor_position: 0
        motor_torque: 0
        position_pid: 0
        velocity_pid: 0
